runtime/include/includej9h.h

```diff
@@ -0,0 +1,51 @@
+/*******************************************************************************
+ * Copyright IBM Corp. and others 2025
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] https://openjdk.org/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-exception-2.0 OR GPL-2.0-only WITH OpenJDK-assembly-exception-1.0
+ *******************************************************************************/
+
+#if !defined(JCLTRACEPOINTS_H)
+#define JCLTRACEPOINTS_H
+
+/* The following definitions are required by j9.h and are provided via OpenJ9
+ * jni.h or the OpenJ9 build system, but needed when building outside of OpenJ9
+ * with j9.h and OpenJDK jni.h is used.
+ */
+struct GCStatus;
+typedef struct GCStatus GCStatus;
+struct JavaVMQuery;
+typedef struct JavaVMQuery JavaVMQuery;
+struct JVMExtensionInterface_;
+typedef const struct JVMExtensionInterface_ *JVMExt;
+
+#define COPY_PROGRESS_INFO_MASK 0
+#if defined(AIX)
+#define AIXPPC
+#define RS6000
+#endif /* defined(AIX) */
+#if defined(MACOSX)
+#define OSX
+#endif /* defined(MACOSX) */
+#if defined(WIN32)
+#define OMR_OS_WINDOWS
+#endif /* defined(WIN32) */
```

Comment:
It's not clear to me that any of this should be necessary.

Reply:
There were all added to fix compilation errors. Do you need me to retry builds without them and record the errors that occur? OpenJ9/OMR use different platform defines than OpenJDK uses/defines. If no matching OpenJ9 platform define is set, there are a couple places that define structures based on platform, and then an error occurs because the structure isn't defined. That was the problem for AIX/Mac, for Windows it may be slightly different but along the same lines.

Reply:
The errors when OSX is not defined below. I think they were the same for AIX but both RS600 and AIXPPC are needed to fix both of these.
```
/Users/jenkins/peter/openj9-openjdk-jdk8/openj9/runtime/oti/j9port.h:82:2: error: "J9PORT_LIBRARY_SUFFIX must be defined"
#error "J9PORT_LIBRARY_SUFFIX must be defined"
 ^
/Users/jenkins/peter/openj9-openjdk-jdk8/openj9/runtime/oti/j9port.h:324:2: error: unknown type name 'SYS_FLOAT'
        SYS_FLOAT cpuEntitlement;
```

Reply:
On Windows
`c:\workspace\openj9-openjdk-jdk8\omr\include_core\omrport.h(55): fatal error C1083: Cannot open include file: 'unistd.h': No such file or directory`